extensions/mssql/src/contextProvider.ts

```diff
@@ -26,6 +27,7 @@ export function setCommandContext(key: ContextKeys | string, value: any) {
 	return vscode.commands.executeCommand(BuiltInCommands.SetContext, key, value);
 }
 
+const isBigDataClusterProperty = 'isBigDataCluster';
```

Comment:
consider moving to constants.ts as I think other files may eventually need this.